### ISSUE TITLE
Issuie plugin WHM - CPanel v66.0.23

### ISSUE THREAD

**Issue by lucianozanita** [Prajithp/letsencrypt-cpanel#111]
Internal Server Error
500
No response from subprocess (/usr/local/cpanel/whostmgr/docroot/cgi/letsencrypt/letsencrypt.pl): The subprocess reported error number 255 when it ended.

How can I solve? already tried to install, uninstall and nothing works

**Comment by photogaff** [comment]
I'm seeing the same issue. also getting "400 Bad request" when I run it through shell.

Running CENTOS 7.4 kvm [cpanel2]  v66.0.23

**Comment by lucianozanita** [comment]
Yes, apparently some incompatibility with the new version of CPanel/WHM

**Comment by robertsimpson** [comment]
I'm experiencing this issue on a new installation too.

**Comment by robertsimpson** [comment]
I noticed this when installing again:

```
Protocol::ACME is NOT installed
Installing Protocol::ACME
.....
.....
 Testing  perl modules we just installed 
.....
Protocol::ACME is installed properly
.....
.....DONE
```

This is despite already having Protocol::ACME installed. Could this be part of the problem? The plugin can't find resources it needs?

**Comment by robertsimpson** [comment]
I just installed the plugin on a completely fresh install of cPanel v66.0.24 and the issue persists.

```
Internal Server Error

500

No response from subprocess (/usr/local/cpanel/whostmgr/docroot/cgi/letsencrypt/letsencrypt.pl): The subprocess reported error number 255 when it ended.

cpsrvd Server at XXX.XXX.XXX.XXX
```

**Comment by photogaff** [comment]
I have abandoned this and moved to the cPanel LetsEncrypt plugin for AutoSSL:
https://documentation.cpanel.net/display/CKB/The+Let%27s+Encrypt+Plugin

Install: /scripts/install_lets_encrypt_autossl_provider
Uninstall: /scripts/uninstall_lets_encrypt_autossl_provider

**Comment by mepjnik** [comment]
Hi,

This fixed my issue:

Login WHM > Tweak Setting > Security > Allow deprecated WHM accesshash authentication > Enable ( On )
